signer/src/block_observer.rs

```diff
@@ -521,7 +522,7 @@ mod tests {
             .unwrap();
         let storage = block_observer.storage.lock().await;
         assert_eq!(storage.deposit_requests.len(), 1);
-        let db_outpoint = (tx_setup0.tx.compute_txid().to_byte_array().to_vec(), 0);
+        let db_outpoint = (BitcoinTxId::from(tx_setup0.tx.compute_txid()), 0);
```

Comment:
I personally think that this syntax is more readable:
```rust
let db_outpoint: (BitcoinTxId, u32) = (tx_setup0.tx.compute_txid().into(), 0);
```
Just because it reduces one level of nesting on the right side (and vscode prints out the types on the left anyway); but the way it is is fine, too.

Reply:
Good point, will update.